### PR TITLE
Add jugend hackt badge

### PR DESCRIPTION
**Pull request by marvinschopf**
Add jugendhackt badge

### CODE REVIEW

**Review by jens1o — DISMISSED** [review]

**Review by jens1o — CHANGES_REQUESTED** [review]
Could you modify the ALT-text to provide a meaningful alternative for impaired users, please?

**Comment by marvinschopf** [comment]
Yep, should fit now. Thanks for the review!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # mkdwv-backend
 ![David](https://img.shields.io/david/Jugendhackt/mkdwv-backend.svg?style=flat-square)
-[![Jugendhackt](https://jhbadge.com/?year=2018&evt=ffm)](https://jugendhackt.org)
+[![Made at Jugend hackt Franfurt 2018](https://jhbadge.com/?year=2018&evt=ffm)](https://jugendhackt.org)
```
Comment (jens1o) [inline README.md]: It's `Frankfurt` ;)

Comment (marvinschopf) [reply README.md]: Fixed :)

**Comment by marvinschopf** [comment]
Fixed :)

**Review by jens1o — APPROVED** [review]

**Pull request merged**